### PR TITLE
Module01 Azure templates basics

### PR DESCRIPTION
**Pull request by dheepdatascigit**

### CODE REVIEW

File: azpshell01deploy.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Deploy the template
+$templateFile = "azuredeploy.json"
+$today=Get-Date -Format "dd-MM-yyyy"
+$deploymentName="addSkuParameterOutputs-"+"$today"
+New-AzResourceGroupDeployment `
+  -Name $deploymentName `
+  -TemplateFile $templateFile `
+  -storageAccountType Standard_LRS `
+  -storageName learnexercisenew334634 `
+  -storageSKU Standard_GRS
```
Comment (netresearchdevops) [inline azpshell01deploy.ps1:10]: Do you need multiple storage parameters

File: azuredeploy.json [hunk 2]
```diff
@@ -2,34 +2,62 @@
...
+           "defaultValue": "Standard_LRS",
+           "allowedValues": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_RAGRS",
+                "Standard_ZRS",
+                "Premium_LRS",
+                "Premium_ZRS",
+                "Standard_GZRS",
+                "Standard_RAGZRS"
```
Comment (netresearchdevops) [inline azuredeploy.json]: To many allowed storage values. Can you reduce it a bit

File: azuredeploy.json [hunk 3]
```diff
@@ -2,34 +2,62 @@
...
     }],
     "outputs": {
-         "storageEndpoint": {
-           "type": "object",
-           "value": "[reference('learnexercise321').primaryEndpoints]"
-       }
+        "storageEndpoint": {
+            "type": "object",
+            "value": "[reference(parameters('storageName')).primaryEndpoints]"
+        }
```
Comment (netresearchdevops) [inline azuredeploy.json]: what is primary endpoints here

File: azpshell01deploy.ps1 (hunk 1 above)
Comment (dheepdatascigit) [reply azpshell01deploy.ps1:10]: Yes, they are different parameters

File: azuredeploy.json (hunk 3 above)
Comment (netresearchdevops) [reply azuredeploy.json]: seems ok.

**Review by netresearchdevops — CHANGES_REQUESTED** [review]
looks ok now

**Review by netresearchdevops — APPROVED** [review]
looks good now

**Pull request merged**